libexec/cli/image.create.exec

```diff
@@ -96,6 +102,19 @@ if [ -f "$SINGULARITY_IMAGE" ]; then
     fi
 fi
 
+# Get the inode option *before* creating the empty file
+# We want to exit before the 'dd' command
+INODE=""
+if [ -n "${SINGULARITY_INODESIZE:-}" ]; then
+    INODE="-i ${SINGULARITY_INODESIZE:-16384}"
+
+    if [ $(expr ${SINGULARITY_INODESIZE} % 1024) -ne 0 ]; then
+        message ERROR "Given inode size of ${SINGULARITY_INODESIZE} is not divisible by 1024\n"
+        exit 1
```

Comment:
Should this be an `ABORT 255`?

Reply:
Sure... I can change that. Call it habit for a `exit 1` failure ...